src/cloudWatchLogs/cloudWatchLogsFiles.ts

```diff
@@ -0,0 +1,15 @@
+import { Uri, Disposable } from 'vscode'
+import { FileProvider, VirualFileSystem } from '../shared/virtualFilesystem'
+import { LogStreamRegistry } from './registry/logStreamRegistry'
+
+export class CloudWatchLogsFileSystem extends VirualFileSystem {
```

Comment:
A VFS is _significantly_ more complex than document providers. Would avoid for now.